assets/locales/service.cy.toml

```diff
@@ -0,0 +1,127 @@
+# active.cy.toml
```

Comment:
Lifted from the localisations in `dp-frontend-renderer`.

Because we now have two localisations files, I suggest renaming the service-specific and renderer lib ones to `service.{language}.toml` and `core.{language}.toml` respectively